cli/cmd/testdata/install_default.golden

```diff
@@ -139,6 +139,7 @@ spec:
     spec:
       containers:
       - args:
+        - linkerd-controller
```

Comment:
i believe that we can just add this to the container image's `ENTRYPOINT` so it needn't be copypasted everywhere

```dockerfile
ENTRYPOINT ["linkerd-controller"]
```